tests/integration/test_helloworld.py

```diff
@@ -13,10 +13,29 @@
 
 
 @pytest.mark.timeout(360)
-def test_helloworld(tmpdir):
+def test_helloworld_call(tmpdir):
```

Comment:
But then why keep this one as well?

Reply:
I thought that it would be better to don't have two asserts in one test, so every test will test other functionality. Now, I don't see any benefit for such approach in this case. I will make one test.